lib/blkproto.mli

```diff
@@ -0,0 +1,122 @@
+val max_segments_per_request : int
+
+module FeatureIndirect : sig
+  type t = { max_indirect_segments : int }
+
+  val _max_indirect_segments : string
+
+  val to_assoc_list : t -> (string * string) list
+end
+
+module State : sig
+  type t = | Initialising | InitWait | Initialised | Connected | Closing | Closed
+  val _state : string
+  val of_string : string -> t option
+  val to_assoc_list : t -> (string * string) list
+  val to_string : t -> string
+end
+
+module Mode : sig
+  type t = ReadOnly | ReadWrite
+end
+
+module Media : sig
+  type t = CDROM | Disk
+end
+
+module DiskInfo : sig
+  type t = {
+    sector_size : int;
+    sectors : int64;
+    media : Media.t;
+    mode : Mode.t;
+  }
+
+  val to_assoc_list : t -> (string * string) list
+end
+
+module Hotplug : sig
+  val _hotplug_status : string
```

Comment:
Is the `_` here intentional?

Reply:
Yes, this is the name given in `blkproto.ml`.  Your guess is as good as mine.

Reply:
Probably some strange naming convention for constant strings before anyone realised that `_` had special meaning?